src/pudl/metadata/enums.py

```diff
@@ -1,97 +1,39 @@
-    "DE": "Delaware",
-    "FL": "Florida",
-    "GA": "Georgia",
-    "HI": "Hawaii",
-    "IA": "Iowa",
-    "ID": "Idaho",
-    "IL": "Illinois",
-    "IN": "Indiana",
-    "KS": "Kansas",
-    "KY": "Kentucky",
-    "LA": "Louisiana",
-    "MA": "Massachusetts",
-    "MD": "Maryland",
-    "ME": "Maine",
-    "MI": "Michigan",
-    "MN": "Minnesota",
-    "MO": "Missouri",
-    "MS": "Mississippi",
-    "MT": "Montana",
-    "NC": "North Carolina",
-    "ND": "North Dakota",
-    "NE": "Nebraska",
-    "NH": "New Hampshire",
-    "NJ": "New Jersey",
-    "NM": "New Mexico",
-    "NV": "Nevada",
-    "NY": "New York",
-    "OH": "Ohio",
-    "OK": "Oklahoma",
-    "OR": "Oregon",
-    "PA": "Pennsylvania",
-    "RI": "Rhode Island",
-    "SC": "South Carolina",
-    "SD": "South Dakota",
-    "TN": "Tennessee",
-    "TX": "Texas",
-    "UT": "Utah",
-    "VA": "Virginia",
-    "VT": "Vermont",
-    "WA": "Washington",
-    "WI": "Wisconsin",
-    "WV": "West Virginia",
-    "WY": "Wyoming",
+from pudl.metadata.dfs import POLITICAL_SUBDIVISIONS
+
+COUNTRY_CODES_ISO3166: set[str] = set(POLITICAL_SUBDIVISIONS.country_code)
+SUBDIVISION_CODES_ISO3166: set[str] = set(POLITICAL_SUBDIVISIONS.subdivision_code)
+EPACEMS_STATES: set[str] = set(
+    POLITICAL_SUBDIVISIONS.loc[
+        POLITICAL_SUBDIVISIONS.is_epacems_state, "subdivision_code"
+    ]
+)
+DIVISION_CODES_US_CENSUS: set[str] = set(
+    POLITICAL_SUBDIVISIONS.division_code_us_census.dropna()
+)
+
+APPROXIMATE_TIMEZONES: dict[str, str] = {
+    x.subdivision_code: x.timezone for x in POLITICAL_SUBDIVISIONS.itertuples()
 }
-"""Mapping of US state abbreviations to their full names."""
+"""Mapping of political subdivision code to the most common timezone in that area.
 
-US_TERRITORIES: dict[str, str] = {
-    "AS": "American Samoa",
-    "DC": "District of Columbia",
-    "GU": "Guam",
-    "MP": "Northern Mariana Islands",
-    "PR": "Puerto Rico",
-    "VI": "Virgin Islands",
-}
-"""Mapping of US territory abbreviations to their full names."""
+This is imperfect for states that have split timezones. See:
+https://en.wikipedia.org/wiki/List_of_time_offsets_by_U.S._state_and_territory
 
-US_STATES_TERRITORIES: dict[str, str] = {**US_STATES, **US_TERRITORIES}
+For states that are split, we chose the timezone with a larger population.
+List of timezones in pytz.common_timezones
+Canada: https://en.wikipedia.org/wiki/Time_in_Canada#IANA_time_zone_database
+"""
 
-EPACEMS_STATES: list[str] = [
-    state
-    for state in US_STATES_TERRITORIES
-    # AK and PR have data but only a few years, and that breaks the Datastore.
-    # See https://github.com/catalyst-cooperative/pudl/issues/1264
-    if state not in {"AK", "AS", "GU", "HI", "MP", "PR", "VI"}
-]
-"""The US states and territories that are present in the EPA CEMS dataset."""
+STATE_TO_CENSUS_DIVISION: dict[str, str] = {
+    x.subdivision_code: x.division_code_us_census
+    for x in POLITICAL_SUBDIVISIONS.itertuples()
+}
```

Comment:
@TrentonBush this is the state to census division (smaller area than census region in the Census nomenclature, even though EIA seems to be calling the census divisions census regions?) that I think you need.